sdk/communication/communication-phone-numbers/samples/v1/typescript/src/updatePhoneNumberCapabilities.ts

```diff
@@ -19,14 +19,17 @@ export async function main() {
 
   // You will need to set this environment variable or edit the following values
   const connectionString =
-    process.env.COMMUNICATION_CONNECTION_STRING ||
+    process.env.AZURE_COMMUNICATION_SAMPLES_CONNECTION_STRING ||
     "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
 
   // create new client
   const client = new PhoneNumbersClient(connectionString);
 
   // You will need to set this environment variable or edit the following values
-  const phoneNumberToUpdate = process.env.PHONE_NUMBER_TO_UPDATE || "<phone number to update>";
+  const phoneNumberToUpdate =
+    process.env.AZURE_COMMUNICATION_PHONE_NUMBER_TO_UPDATE ||
```

Comment:
I really like the naming convention as AZURE_COMMUNICATION_SAMPLES_CONNECTION_STRING, and i like how that simplifies it to 1 env var.
I wish there was something we could do for the phone number but I can't think of anything better